### DISCUSSION TITLE
Translation and modifications for Swedish users (other languages after modification)

### DISCUSSION THREAD

**Discussion by Svardsten53** [mcguirepr89/BirdNET-Pi#391] [Show and tell]
Originally for myself, but later also for others, I have made a package with translation into Swedish and some other modifications of BirdNET-Pi. I have had great help from Patrick with the git stuff.

- Translation into Swedish of all pages except for the ones under Tools.
 
- A more customized heading where the location and the country's flag are included.
 
- A daily chart on the overview page where ALL of today's species are included, even the most interesting at the bottom of the list.
 
- External links adapted for Swedish species.
 
- Corrected Swedish species list.
 
- Page with links added that you can easily modify yourself.
 
- Easy to exclude modifications you do not want or if you want to change translations.
 
- Complete documentation.

The package is based on a number of sed- and copy commands. I have limited experience with Linux and coding, so it's a semi-automated routine, not fancy in any way, but it works. I have tested it several times myself and also on some other users installations.

The instructions are written in Swedish (better good Swedish than bad English for Swedish users), but everything can be customized to other languages ​​with a little effort, Google Translate is your friend.

I have also created an installation instruction, also including how to build a microphone and where to buy things, and a user manual, everything in Swedish.

Feel free to look at my page what it looks like and if you are interested in the package, you can email the address at the bottom of the link page - "Länkar". I will try to keep the package updated and distribute it via a download link.

https://birdnet.svardsten.se/

**Comment by Svardsten53** [comment]
Please check that your email address is correct. One of the answers I received has an address that can not be sent to.

**Comment by krummrey** [comment]
I guess having a translatable UI would be nice to have. Into any language.

**Comment by Oripy** [comment]
Hello !
How did you do the translation? I'm interested in translating the "user" pages into French.

**Reply by Svardsten53** [reply]
For the traslation I'm using a bunch of sed commands in a script.
Example: `sed -i 's/No Detections For Today./Inga fynd i dag än/g' overview.php`

If you want to know more please send an email to birdnetpi@svardsten.com
